HandyImage.user.js

```diff
@@ -796,17 +795,19 @@ function makeworld()
 	{
 	case "gist.github.com":
 		if(document.body){i=1;cfg();}break;
-	case "vvcap.net":
 	case "simplest-image-hosting.net":
 	case "hostimage.ru":
 	case "imagebin.ca":
 	case "imgchili.net":
-	case "pixhst.com":
 	case "adultimages.xyz":
-	case "depic.me":
 		i = q('img');
+		if(i)
```

Comment:
@Owyn I added this code because old code didn't find the full image.
Would this code make other sites load longer? (Sorry, this is the first time that I am using .replace function)